src/posit/connect/groups.py

```diff
@@ -10,18 +10,83 @@
 if TYPE_CHECKING:
     import requests
 
+    from posit.connect.context import Context
+
+    from .users import User
+
 
 class Group(Resource):
+    def __init__(self, ctx: Context, **kwargs) -> None:
+        super().__init__(ctx.client.resource_params, **kwargs)
+        self._ctx: Context = ctx
+
+    @property
+    def members(self) -> GroupMembers:
+        """Get the group members.
+
+        Returns
+        -------
+        GroupMembers
+            All the users in the group.
+
+        Examples
+        --------
+        ```python
+        group = client.groups.get("GROUP_GUID_HERE")
+        group_users = group.members.find()
+        group_user_count = group.members.count()
+        ```
+
+        """
+        return GroupMembers(self._ctx, group_guid=self["guid"])
+
     def delete(self) -> None:
         """Delete the group."""
         path = f"v1/groups/{self['guid']}"
         url = self.params.url + path
         self.params.session.delete(url)
 
 
+class GroupMembers(Resources):
+    def __init__(self, ctx: Context, group_guid: str) -> None:
+        super().__init__(ctx.client.resource_params)
+        self._group_guid = group_guid
+        self._ctx: Context = ctx
+
+    def find(self) -> list[User]:
+        # Avoid circular import
+        from .users import User
```

Comment:
I wonder if this is a sign that we actually want users and groups in the same file? That would be a much larger change of course, but curious your thoughts on that

Reply:
I don't mine the separation because of the strong type support. 

If there's more situations like this, we can think on moving them together later.

(Note, we'll need to maintain the `groups.py` and `users.py` files if we're to maintain the exported objects. But they can totally be _implemented_ in another file, e.g. `_usergroups.py`)